.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,8 +12,16 @@
 Add one of the following:
 bug
 cleanup
+dependency update
 documentation
 feature
+improvement
+release workflow
+testing
+-->
+
+<!--
+If this test will only be compatible with a future VPC CNI version release, please add a Skip so that automated testing can skip it
```

Comment:
nit: this comment belongs in the testing section, and maybe we can reword to:
```
If adding a new integration test to any of the CNI release test suites, determine if the test can run against the latest VPC CNI image or if it is dependent on a future version release. If dependent, please call `Skip()` in the test to prevent it from running before the future version is available.
```